content/10.diagnostics.md

```diff
@@ -144,51 +144,51 @@ In some cases, library preparation can involve other modifications of the sample
+One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6].
 This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
 The authors break down the sensitivity of their test according to the C~t~ value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less.
 The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
 However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool.
-Various combinations of reagents are available, but one example of reagents being used is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].
-To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed _N_-gene RNA standard (IVT RNA) were tested using quantities from 10^5 copies down to 10 copies.
+Various combinations of reagents are available, but one example is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].
+To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed _N_-gene RNA standard were tested using quantities from 10^5^ copies down to 10 copies.
 The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes.
 The RT-LAMP assay was then applied to clinical nasopharyngeal samples.
 For viral loads above 100 copies of genomic RNA, the RT-LAMP assay had a sensitivity of 100% and a specificity of 96.1% from purified RNA.
 The direct assay of saliva by RT-LAMP had a sensitivity of 85% (CI 70-93%).
 Sensitivity and specificity metrics were obtained by comparison with results from RT-PCR.
-RT-LAMP pilot studies for detection of SARS-CoV-2 were recently reviewed in a meta-analysis [@PMID:34086539].
-In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was 95.5% (CI 97.5% = 90.8-97.9%) and the cumulative specificity was 99.5% (CI 97.5% = 97.7-99.9%).
+RT-LAMP pilot studies for detection of SARS-CoV-2 were reviewed in a meta-analysis [@PMID:34086539].
+In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was 95.5% (CI 97.5% = 90.8-97.9%) and the cumulative specificity was 99.5% (CI 97.5% = 97.7-99.9%). <!-- This is different confidence interval notation than we've used elsewhere -->
```

Comment:
Grep on "CI" suggests we usually don't mention them (I only got 4 lines) so I am going to make this less precise.

```suggestion
In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was calculated at 95.5%, and the cumulative specificity was 99.5%.
```